### PR TITLE
golang/Cloud Run: Refine .vscode, README, and enable in .template

### PR DESCRIPTION
**Pull request by grayside**

### CODE REVIEW

File: golang/go-cloud-run-hello-world/.vscode/launch.json [hunk 1]
```diff
@@ -1,21 +1,23 @@
...
-      "name": "Run on Cloud Run",
+      "name": "Run as Cloud Run Service Locally",
       "type": "cloudcode.cloudrun",
       "request": "launch",
-      "skaffoldConfig": "${workspaceFolder}/skaffold.yaml",
+      "service": {
+        "name": "go-cloud-run-hello-world"
+      },
+      "build": {
+        "image": "gcr.io/adamross-svls-kibble/go-cloud-run-hello-world",
```
Comment (simonz130) [inline golang/go-cloud-run-hello-world/.vscode/launch.json]: Remove your private repo reference from the template.

File: golang/go-cloud-run-hello-world/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Contributing
+
+Join us at https://github.com/GoogleCloudPlatform/cloud-code-samples.
+
+## Run the Tests
```
Comment (simonz130) [inline golang/go-cloud-run-hello-world/CONTRIBUTING.md]: Same comment as in nodejs - this info should be at the root CONTRIBUTING.md

Comment (grayside) [reply golang/go-cloud-run-hello-world/CONTRIBUTING.md]: This has sample-specific instructions that will be even less generic once we centralize cloudbuild.yaml by language. I guess we could have a CONTRIBUTING per language and list per sample instructions?

Comment (simonz130) [reply golang/go-cloud-run-hello-world/CONTRIBUTING.md]: yep, that was the intent

Comment (simonz130) [reply golang/go-cloud-run-hello-world/CONTRIBUTING.md]: yes

**Comment by grayside** [comment]
I'll merge this as-is and follow up with CONTRIBUTING text improvements and CI adjustments tomorrow.

**Pull request merged**